ManagerQueue/Queue.cs

```diff
@@ -0,0 +1,157 @@
+                return false;
+            }
+        }
+    }
+
+    public class QueueTransit
+    {
+        public Subscription SubscriptionForFirstQueue;
+        public Subscription SubscriptionForSecondQueue;
+        public Queue FirstQueue;
+        public Queue SecondQueue;
+        public bool ServiceWork = true;
+
+        public QueueTransit(string configForFirstQueue, string configForSecondQueue)
+        {
+            this.FirstQueue = new Queue(configForFirstQueue);
+            this.SubscriptionForFirstQueue = new Subscription(FirstQueue.Model, FirstQueue.QueueName, false);
+            this.SecondQueue = new Queue(configForSecondQueue);
+            this.SubscriptionForSecondQueue = new Subscription(SecondQueue.Model, SecondQueue.QueueName, false);
+        }
+
+        public void TransitData()
+        {
+            try
+            {
+                var basicDeliveryEventArgs = SubscriptionForFirstQueue.Next();
+                var messageContent = Encoding.UTF8.GetString(basicDeliveryEventArgs.Body);
+                if (messageContent == "stop")
+                {
+                    SubscriptionForFirstQueue.Ack(basicDeliveryEventArgs);
+                    FirstQueue.Model.Close();
+                    FirstQueue.Connection.Close();
+                    SubscriptionForFirstQueue.Close();
+                    SendToTwoQueue(messageContent);
+                    return;
+                }
+
+                Console.WriteLine($"Data: '{messageContent}' send to Queues #2");
+                SubscriptionForFirstQueue.Ack(basicDeliveryEventArgs);
+                SendToTwoQueue(messageContent);
+            }
+            catch (Exception exception)
+            {
+                Console.WriteLine(exception.ToString());
+            }
+        }
+
+        private void SendToTwoQueue(string data)
+        {
+            SecondQueue.SetData(data);
+        }
+
+        public void GetData()
+        {
+            try
+            {
+                var basicDeliveryEventArgs = SubscriptionForSecondQueue.Next();
+                var messageContent = Encoding.UTF8.GetString(basicDeliveryEventArgs.Body);
+                if (messageContent == "stop")
+                {
+                    SubscriptionForSecondQueue.Ack(basicDeliveryEventArgs);
+                    SubscriptionForSecondQueue.Close();
+                    FirstQueue.Model.Close();
+                    FirstQueue.Connection.Close();
+                    ServiceWork = false;
+                    return;
+                }
+                Console.WriteLine("Get data from queue #2: " + messageContent);
+                SubscriptionForSecondQueue.Ack(basicDeliveryEventArgs);
+            }
+            catch (Exception exception)
+            {
+                Console.WriteLine(exception);
+            }
+        }
+
+        public bool ServiceOperability()
+        {
+            if (ServiceWork == true) return true;
```

Comment:
some times it’s hard to see if return on the same row as if statement, especially if else block is present